### PR TITLE
DPPA-95 image_id() returns incorrect error when no results found

### PR DESCRIPTION
**Pull request by stanislaw-ionpath**
image_id() returns incorrect error when no results found

### CODE REVIEW

File: mibitracker/request_helpers.py [hunk 1]
```diff
@@ -475,13 +475,16 @@ def image_id(self, run_name, point_name):
         ).json()
 
         try:
-            assert len(results) == 1
+            len_results = len(results)
```
Comment (rfinck) [inline mibitracker/request_helpers.py]: This'll work, although is there a need to keep the TypeError clause or can we go straight to checking the length?

Comment (stanislaw-ionpath) [reply mibitracker/request_helpers.py]: @rfinck I don't know what is actually returned by the API call. Where can I check that?

Comment (rfinck) [reply mibitracker/request_helpers.py]: You can make a request to our instance and see what you get back? (Also, please edit the name of this PR to have a brief description in addition to the ticket number.)

Comment (stanislaw-ionpath) [reply mibitracker/request_helpers.py]: @rfinckit seems we do not need TypeError.

Comment (rfinck) [reply mibitracker/request_helpers.py]: no one has ever called me rfinckit before ...

if we don't need the TypeError, please remove it and then we can merge this it.

Comment (stanislaw-ionpath) [reply mibitracker/request_helpers.py]: @rfinck done

File: mibidata/mibi_image.py [hunk 2]
```diff
@@ -261,10 +261,11 @@ def channel_inds(self, channels):
         except (ValueError, AttributeError):
             if self.targets is None:
                 error_msg = f'Channels were indexed with targets only (no ' \
-                    f'masses were given), available targets are {channels}'
+                    f'masses were given), available targets are ' \
+                    f'{self._channels}'
```
Comment (jaytarolli) [inline mibidata/mibi_image.py]: Sorry if I missed this, but what is the purpose of changing from `channels` to `self._channels` here?

Comment (stanislaw-ionpath) [reply mibidata/mibi_image.py]: This is a separate bug that I noticed. 

`self._channles` are channels available in the mibiimage, `channels` are channels that the function should return.

Comment (jaytarolli) [reply mibidata/mibi_image.py]: The way it was written made sense to me, so maybe I was misinterpreting it all along. The new error message added here, as you have written it, makes sense. But `Subset of channels, targets or massses not found matching ...` makes sense with the channels that were passes as an argument and not the full list of channels in the image.

**Pull request merged**